File: 프로그래머스/unrated/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.cs
using System;

public class Solution {
    public string solution(string my_string, int[,] queries) 
    {
        var arr = my_string.ToCharArray();
        for(int y = 0; y < queries.GetLength(0); ++y)
        {
            int s = queries[y, 0];
            int e = queries[y, 1];
            
            Array.Reverse(arr, s, e - s + 1);
        }
        return new string(arr);
    }
}

File: 프로그래머스/0/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string, vector<vector<int>> queries) {
        
    for (int i = 0; i < queries.size(); ++i)
    {
        int from = queries[i][0];
        int end = queries[i][1];
        
        std::reverse(my_string.begin() + from, my_string.begin() + end + 1);
    }
        
    return my_string;
}